composer.json

```diff
@@ -29,6 +29,7 @@
     },
     "require": {
         "php": ">=7.2.0",
+        "ext-json": "*",
```

Comment:
Would be nice if these stayed alphabetical.

Reply:
I dont think thats a big deal, but you can sure reorder them.